app/views/vendors/index.html.erb

```diff
@@ -1,76 +1,45 @@
 <div class="row">
-  <% @vendors.each do |vendor| %>
-
   <div class="large-4 small-12 columns">
 
     <img src="http://placehold.it/500x500&text=Logo">
 
     <div class="hide-for-small panel">
-      <h3>Header</h3>
-      <h5 class="subheader">Risus ligula, aliquam nec fermentum vitae, sollicitudin eget urna. Donec dignissim nibh fermentum odio ornare sagittis.
+      <h3>Vendors</h3>
+      <h5 class="subheader">This is an index of all available vendors.
       </h5>
     </div>
 
+    <div>
+      <% if user_signed_in? %>
+        <h5>Add a Vendor</h5>
+        <%= form_for @new_vendor do |f| %>
+
+        <%= f.label :vendor_name, 'Vendor Name' %>
+        <%= f.text_field :vendor_name %>
+
+        <%= f.submit "Add Vendor" %>
+        <% end %>
```

Comment:
It would be cool if this form was in a [partial](http://guides.rubyonrails.org/layouts_and_rendering.html#using-partials) under the `app/assets/views/vendors` folder.